### ISSUE TITLE
is it possible to use only one button?

### ISSUE THREAD

**Issue by memesanddremes** [naelstrof/VRPlayspaceMover#22]
I would like to use this but in one of my games i need to use the A button which can get annoying since it keeps moving my play space. is there a way to only use the x button for movement?

**Comment by naelstrof** [comment]
Yes! You may change the bindings using instructions in the README.md.
